### PR TITLE
Migrate the CI pre-merge tests from eve to GitHub actions

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
- Support new tox 4.0 version
- Disable eve pre-merge
- Move the build to GitHub actions
- Move the e2e tests to GitHub Actions (in progress => single node done but not tested yet)
- Move integration tests UI to GitHub Actions

### CODE REVIEW

**Review by JBWatenbergScality — APPROVED** [review]

File: images/metalk8s-alert-logger/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ COPY main.go go.mod "$PKG_PATH"
 WORKDIR "$PKG_PATH"
 
 RUN sed -i "s/@@ALERTMANAGER_VERSION@@/$ALERTMANAGER_VERSION/g" go.mod \
-  && go mod tidy \
+  && go mod tidy -go=1.16 && go mod tidy -go=1.17 \
```
Comment (TeddyAndrieux) [inline images/metalk8s-alert-logger/Dockerfile:21]: Why do we need both ? (@JBWatenbergScality )

**Pull request merged**